.devcontainer/devcontainer.json

```diff
@@ -9,20 +9,33 @@
       "**/.git/subtree-cache/**": true,
       "target/**": true
     },
-    "dprint.path": "/home/vscode/.dprint/bin/dprint"
+    "[jsonc]": {
+      "editor.defaultFormatter": "esbenp.prettier-vscode"
+    },
+    "[yaml]": {
+      "editor.defaultFormatter": "esbenp.prettier-vscode"
+    },
+    "[typescript]": {
+      "editor.defaultFormatter": "dprint.dprint"
+    },
+    "deno.codeLens.testArgs": ["--no-check=remote", "-A", "-L=info"],
+    "deno.config": "./deno.jsonc",
+    "deno.enable": true,
+    "deno.lint": true,
+    "editor.defaultFormatter": "dprint.dprint",
+    "editor.formatOnSave": true,
+    "editor.tabSize": 2,
+    "prettier.printWidth": 100
   },
   "extensions": [
     "EditorConfig.EditorConfig",
     "bungcip.better-toml",
```

Comment:
We can get rid of this as well, as I believe we no longer have any toml in this repo?

Reply:
There is a `book.toml` file.
Shall I remove it anyway?